cypress/e2e/article.cy.js

```diff
@@ -1,24 +1,81 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
-describe('Article', () => {
-  before(() => {
+import ArticlePageObject from '../support/pages/article.pageObject';
+import HomePageObject from '../support/pages/home.pageObject';
+import SignInPageObject from '../support/pages/signIn.pageObject';
 
-  });
+const articlePage = new ArticlePageObject();
+const homePage = new HomePageObject();
+const signInPage = new SignInPageObject();
 
-  beforeEach(() => {
+describe('Article', () => {
+  let user;
+  let article;
+
+  before(() => {
     cy.task('db:clear');
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
+    cy.task('generateArticle').then((generateArticle) => {
+      article = generateArticle;
+    });
   });
 
   it('should be created using New Article form', () => {
+    signInPage.visit();
+    cy.register(user.email, user.username, user.password);
+
+    signInPage.typeEmail(user.email);
+    signInPage.typePassword(user.password);
+    signInPage.clickSignInBtn();
```

Comment:
This can be moved in beforeEach
- not related to test itself (preconditions)
- duplicated in each test